ql/src/java/org/apache/hadoop/hive/ql/ddl/DDLTask.java

```diff
@@ -90,12 +92,16 @@ public int execute() {
         LOG.warn("Error while table creation: ", e);
         return 0;
       }
+      int excpErrorCode = ReplUtils.handleException(work.isReplication(), e, work.getDumpDirectory(),
```

Comment:
could you please rename var to simply `errorCode` and format the code: there should be space between `if` and opening `(`